lib/matplotlib/backends/backend_pgf.py

```diff
@@ -48,12 +48,16 @@ def get_fontspec():
         families = ["serif", "sans\\-serif", "monospace"]
         commands = ["setmainfont", "setsansfont", "setmonofont"]
         for family, command in zip(families, commands):
-            # 1) Forward slashes also work on Windows, so don't mess with
-            # backslashes.  2) The dirname needs to include a separator.
-            path = pathlib.Path(fm.findfont(family))
-            latex_fontspec.append(r"\%s{%s}[Path=\detokenize{%s}]" % (
-                command, path.name, path.parent.as_posix() + "/"))
-
+            try:
+                # 1) Forward slashes also work on Windows, so don't mess with
+                # backslashes.  2) The dirname needs to include a separator.
+                path = pathlib.Path(fm.findfont(family,
+                                                fallback_to_default=False))
+                latex_fontspec.append(r"\%s{%s}[Path=\detokenize{%s}]" % (
```

Comment:
Very minor style point: I prefer the style at #20945 (moving the `append` to an `else` block) as it makes it clearer that it's catching a potential ValueError from findfont.

Reply:
OK! I'm not that well versed in else with try (and for). When I get back to this (and have some idea how to actually get it to work..), I'll change it.